src/olympia/devhub/tests/test_views.py

```diff
@@ -3835,3 +3835,29 @@ def test_devhub_version_edit_page(self):
     def test_devhub_version_list_page(self):
         url = reverse('devhub.addons.versions', args=[self.addon.slug])
         self.assertNameAndNoXSS(url)
+
+
+class TestSwitchToUnlisted(UploadTest):
```

Comment:
well, it's a one line change to `TestVersion.test_user_can_unlist_addon` and 'TestVersion.test_user_can_unlist_hidden_addon` of `assert file.status == amo.STATUS_PUBLIC` or all these lines being added here :)